listener/upload_listener.py

```diff
@@ -360,7 +365,12 @@ def process_upload(upload_data, loop=None):
                     "request_id": upload_data['platform_metadata']["request_id"]
                 },
                 "timestamp": upload_data["timestamp"]}
-            EVALUATOR_QUEUE.send(new_upload_msg, loop=loop)
+            if VmaasStatus.VMAAS_REFRESHING:
+                metadata = EVALUATOR_QUEUE.populate_batch(new_upload_msg)
```

Comment:
it's caching messages in app, what's capacity of the batch? wouldn't be simplier to just stop/pause consumer during this period?

also why are we handling it in listener? I think it can be safely written to evaluator queue and handle it in evaluator, listener doesn't interact with vmaas